en/docs/learn/examples/protocol-switching/switching-between-http-and-msmq.md

```diff
@@ -36,15 +34,13 @@ In this example, stockquote requests are placed to the stockquote proxy service,
             <inSequence>
                <property name="FORCE_SC_ACCEPTED"  value="true"  scope="axis2"  type="STRING"/>
                <property name="OUT_ONLY" value="true" scope="default" type="STRING"/>
+               <log level="custom">
+                <property name="MESSAGE" value="OUT SEQENCE CALLED"/>
```

Comment:
Let's change the log message. We are not calling out sequence here.

Reply:
Fixed